flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/JdbcTestFixture.java

```diff
@@ -149,7 +149,7 @@ private static String getCreateQuery(String tableName) {
         return "CREATE TABLE "
                 + tableName
                 + " ("
-                + "id INT NOT NULL DEFAULT 0,"
+                + "id INT DEFAULT 0 NOT NULL,"
```

Comment:
I have to changes this, to work in oracle.. 
otherwise this error is throw: "ORA-00907: missing right parenthesis"